lib/Notification/Notifier.php

```diff
@@ -297,6 +300,27 @@ protected function shortenJsonEncodedMultibyteSave(string $subject, int $dataLen
 		return $temp;
 	}
 
+	protected function parseStoredRecordingFail(
+		INotification $notification,
+		Room $room,
+		Participant $participant,
+		IL10N $l
+	): INotification {
+		$notification
+			->setRichSubject(
+				$l->t('Fail to store recording of call {call}, reach out to the admin.'),
```

Comment:
```suggestion
				$l->t('Failed to store recording of call {call}, reach out to the admin.'),
```

Not sure if it's correct, but I would go with "Failed" here?!

Reply:
Thanks! I will push when solve the question about type.